optuna_dashboard/_preferential_history.py

```diff
@@ -43,38 +54,57 @@ def report_history(
     study_id: int,
     storage: BaseStorage,
     input_data: NewHistory,
-) -> None:
+) -> str:
     preferences = []
     # TODO(moririn): Use TypeGuard after adding other history types.
     if input_data.mode == "ChooseWorst":
         preferences = [
-            (best, input_data.clicked)
-            for best in input_data.candidates
-            if best != input_data.clicked
+            (better, input_data.clicked)
+            for better in input_data.candidates
+            if better != input_data.clicked
         ]
     else:
         assert False, f"Unknown data: {input_data}"
 
-    preference_id = report_preferences(
+    id = report_preferences(
```

Comment:
Could you avoid using `id`?

```suggestion
    preference_id = report_preferences(
```